apps/website/src/components/Playground/index.tsx

```diff
@@ -2,10 +2,11 @@ import React from 'react';
 import { SandpackProvider, SandpackLayout, SandpackCodeEditor, SandpackPreview } from '@codesandbox/sandpack-react';
 import { useColorMode } from '@docusaurus/theme-common';
 import { useLocation } from '@docusaurus/router';
+
 import AppCode from '!!raw-loader!./code/app.js';
 import DefaultStylesCode from '!!raw-loader!./code/styles.js';
 
-const ctx = require.context('!!raw-loader!./code/templates', false, /\.js$/);
+const ctx = require.context('./code/templates', false, /\.js$/);
```

Comment:
`raw-loader` is not needed, now handled to Webpack's config.

Reply:
why is raw-loader not needed in `require.context` but still needed in the `import` statements ?

Reply:
I added overrides only for `/code/templates`, will added them for other files.